tla-types/src/main/scala/at/forsyte/apalache/tla/typecmp/raw/RawArithmeticBuilder.scala

```diff
@@ -0,0 +1,49 @@
+package at.forsyte.apalache.tla.typecmp.raw
+
+import at.forsyte.apalache.tla.lir.TlaEx
+import at.forsyte.apalache.tla.lir.oper.TlaArithOper
+
+/**
+ * Raw builder for TlaArithOper expressions.
+ *
+ * @author
+ *   Jure Kukovec
+ */
+trait RawArithmeticBuilder extends ProtoBuilder {
```

Comment:
```suggestion
trait UnsafeArithmeticBuilder extends ProtoBuilder {
```

Again, "raw" does not tell much to the reader. I would call it "unsafe", because it may construct ill-typed expressions. Hence, the API user should not even look at this class.